### PR TITLE
[TP-3] Connected

### PR DESCRIPTION
**Pull request by Shapovall**
Full connect express application to database

### CODE REVIEW

**Review by StayTeeVone — DISMISSED** [review]
GOOD!!!

File: config/pepperTeamAPI.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//connect
+db.connect((err) => {
+    if (err){
+        console.log(err);
+        console.error('PepperDB was not connected...');
+    } else
+    console.log('PepperDB connected...')
+});
+
+module.exports = db;
```
Comment (buba3230) [inline config/pepperTeamAPI.js:20]: Add empty line in the end if file

**Review by OleynikBogdan — COMMENTED** [review]
nice

**Review by OleynikBogdan — DISMISSED** [review]

File: routes/city.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { request } = require("express");
+var express = require("express");
+var router = express.Router();
+
+var db = require("../config/pepperTeamAPI");
+
+module.exports = router;
```
Comment (buba3230) [inline routes/city.js:7]: And here too

**Review by buba3230 — CHANGES_REQUESTED** [review]

**Comment by buba3230** [comment]
And what about to add .gitignore ? And remove all libraries from repository...

**Pull request closed without merging**